### PR TITLE
Stimulus controllers needed to be default exports.

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fixes a bug caused by #12512. It was not caught during a smoke test of the admin section.

## Related Tickets & Documents

- #12512 
- #12541

## QA Instructions, Screenshots, Recordings

1. Log in as an administrator in your local development environment. email: admin@forem.local, password: password.

2. Ensure that each admin page has a controller associated to it. The best way to test this is to expand the configuration section, e.g. Authentication and click on a button in it. It should work like before instead of saying that a field is required for the message you need to enter to submit the form. i.e. 👇🏻

image

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added tests?

- [ ] Yes
- [x] No, and this is why: The imports were not correct. A test for this does not make sense. An end to end test of the entire admin section makes sense, but we're not there yet. For now a manual test is best.
- [ ] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Fry from Futurama saying fix it

### CODE REVIEW

File: app/javascript/admin/__tests__/controllers/article_controller.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Application } from 'stimulus';
-import { ArticleController } from '../../controllers/article_controller';
+import ArticleController from '../../controllers/article_controller';
```
Comment (nickytonline) [inline app/javascript/admin/__tests__/controllers/article_controller.test.js:2]: All Stimulus controllers need to be default exports. The new eslint rule we added did not take this into account and we missed this when smoke testing #12512.

File: app/javascript/admin/__tests__/controllers/data_update_script_controller.test.js [hunk 2]
```diff
@@ -46,13 +46,19 @@ describe('DataUpdateScriptController', () => {
       const button = document.getElementById('data_update_script_1_button');
       button.click();
 
-      expect(document.getElementById('data_update_script_1_run_at').innerHTML).toMatch(/loading/);
-      expect(document.getElementById('data_update_script_1_status').innerHTML).toEqual("");
+      expect(
```
Comment (nickytonline) [inline app/javascript/admin/__tests__/controllers/data_update_script_controller.test.js:49]: Some prettier formatting going on here.

File: app/javascript/admin/__tests__/controllers/data_update_script_controller.test.js [hunk 3]
```diff
@@ -61,32 +67,38 @@ describe('DataUpdateScriptController', () => {
...
+        },
+      };
       fetch.mockResponse(JSON.stringify(response));
 
       const button = document.getElementById('data_update_script_1_button');
       button.click();
 
       const flushPromises = () => new Promise(setImmediate);
-      Promise.resolve().then(() => jest.advanceTimersByTime(1000));
+      await Promise.resolve().then(() => jest.advanceTimersByTime(1000));
```
Comment (nickytonline) [inline app/javascript/admin/__tests__/controllers/data_update_script_controller.test.js:93]: eslint was complaining about this, so I fixed it.

**Comment by nickytonline** [comment]
@msarit and @katiedavis, please take this for a test drive. All Stimulus controllers are now default exports, so things should behave as they did prior to #12512.

**Review by rhymes — APPROVED** [review]

**Review by msarit — APPROVED** [review]
Nice work!

**Review by katiedavis — APPROVED** [review]

**Pull request merged**